CONTRIBUTING.md

```diff
@@ -361,143 +254,105 @@ $ git log master.. --oneline --decorate=no
 60e2b2b doh - missed one
 ```
 
-> **Note:** The `git log` command compares your current branch (`fix-doc-bugs`)
-> with the `master` branch and lists all the commits, one per line.
-
-Since all four commits are related to *the same change* to fix spelling mistakes
-and break long lines up into shorter lines, it makes sense to combine all four
-commits into a *single commit* on your PR. To do this, complete the following
-steps:
-
-- Update your branch
-
-  You need to ensure your local copy of the upstream branch contains the
-  latest changes.
+**Note:** The `git log` command compares your current branch (`fix-contrib-bugs`) with the `master` branch and lists all the commits, one per line.
 
-- Rebase your changes against the `master` branch
+##### Git rebase if multiple commits
 
-  This operation might seem scary, but it is the way Git allows you to merge
-  all of your changes together.
+Since all four commits are related to *the same change*, it makes sense to
+combine all four commits into a *single commit* on your PR. You need to
+[git rebase](https://help.github.com/en/github/using-git/about-git-rebase)
```

Comment:
i18n point: Although GitHub doesn't appear to offer help in many languages, it *does* offer it in Chinese fwics:

- https://help.github.com/zh/github/using-git/about-git-rebase

Since GitHub appears to auto-redirect based on the browser locale, we might want to consider dropping the hard-coded two-character language code ("en") for all GitHub URLs in this doc to allow the user to get the official help docs in their language (if possible - it seems to default to English). In other words, the URL written above would be:

- https://help.github.com/github/using-git/about-git-rebase

Reply:
Duly noted. Revised to remove `en` or `zh` going forward.